server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryLifecycle.java

```diff
@@ -0,0 +1,37 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+import java.io.IOException;
+
+import org.opentosca.toscana.core.transformation.TransformationContext;
+import org.opentosca.toscana.plugins.lifecycle.AbstractLifecycle;
+
+public class CloudFoundryLifecycle extends AbstractLifecycle {
+
+    private final TransformationContext context;
```

Comment:
This is not needed, because the superclass stores the context too.

```java
public abstract class AbstractLifecycle implements TransformationLifecycle {

    /**
     The transformation specific logger that can be used to log to the transformations Log Object
     */
    protected Logger logger;
    /**
     The context on which the transformation should be transformed
     */
    protected TransformationContext context;

    /**
     @param context because the context is always needed this should never be null
     It probably gets called by the <code>getInstance</code> method of the LifecycleAwarePlugin
     */
    public AbstractLifecycle(@NotNull TransformationContext context) throws IOException {
        this.context = context;
        this.logger = context.getLogger(getClass());
```
Looking at this we could propbably make the fields final... Will open a issue for that